lib/perl/Genome/Interfaces/Comparable/Command/Diff.pm

```diff
@@ -58,6 +58,7 @@ sub diffs_message {
     my $self = shift;
     my $diffs = $self->_diffs;
     my $diff_string = "DIFFERENCES FOUND:\n";
+    $diff_string .= sprintf(" Comparing new object %s to blessed object %s\n", $self->new_object->id, $self->blessed_object->id);
```

Comment:
It's a little strange that executing this command will print this line twice if there are diffs.